README.md

```diff
@@ -1,3 +1,59 @@
 # Kinto React Boilerplate
 
 A sample [Kinto](https://github.com/mozilla-services/kinto.js) & [React](http://facebook.github.io/react/) project boilerplate.
+
+## Scope
+
+The main purpose of this boilerplate is to speed up the conception of a Kinto JavaScript application, with basic setup for unit and functional testing. It comes with:
```

Comment:
"Conception" has a slightly different meaning in English; in general, I would use "development" instead. For example:
> The boilerplate helps you to develop Kinto JavaScript applications faster by making some decisions on your behalf. It includes: